deptofdefense/move.mil
# Locator Maps v4

## Checklist

I have…


## Summary of Changes

This pull request refactors location-based searching to use Google's reverse geocoding features (via the [Geokit](https://github.com/geokit/geokit) gem). For users, this should improve the experience of searching for things like "Fort Belvoir," "Ft. Belvoir," "San Antonio," "90210," or just about anything else that Google can comprehend.

As a result, we're now able to remove the `Installation` and `ZipCodeTabulationArea` models (and their associated specs, fixtures, seed data, etc.).

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. `git checkout locator-maps-v4`,
1. set up development dependencies according to `CONTRIBUTING.md`,
1. run `bin/rails server`,
1. load up http://localhost:3000/resources/locator-maps, perform a search, and check out those search results!
